BUILD.gn

```diff
@@ -887,6 +887,18 @@ group("licenses") {
   ]
 }
 
+copy("electron_mksnapshot") {
+  sources = []
+  if (is_win) {
+    sources += [ "mksnapshot.exe" ]
+  } else {
+    sources += [ "mksnapshot" ]
```

Comment:
I'm not sure this is sufficient? It looks like `mksnapshot` might depend on `icudtl.dat`:

```
$ gn desc out/Release v8:mksnapshot runtime_deps
./mksnapshot
icudtl.dat
```

We might be better off patching v8/BUILD.gn to expose the `mksnapshot` target so that we can depend on that directly.

Reply:
@nornagon We haven't distributed `icudtl.dat` in previous releases, eg see: https://github.com/electron/electron/releases/download/v3.0.0/electron-v3.0.0-linux-x64.zip

Reply:
@nornagon decided to just patch v8's `BUILD.gn`